rpc/src/types/unistring.rs

```diff
@@ -0,0 +1,30 @@
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+#[serde(untagged)]
+pub enum Unistring {
+  ValidUtf8(String),
+  InvalidUtf8 { invalid_utf8_string: Vec<u8> },
+}
+
+impl Unistring {
+  pub fn get_string(&self) -> String {
+    match self {
+      Self::ValidUtf8(valid_utf8) => valid_utf8.to_string(),
+      Unistring::InvalidUtf8 {
```

Comment:
Shouldn't this be `Self::InvalidUtf8` rather than `Unistring::InvalidUtf8`?

Reply:
Whoops, I was testing some stuff out and I didn't noticed this before commit, will be changed